### PR TITLE
fixing placementeditor try/catch block

### PR DESCRIPTION
**Pull request by Hazzamanic**
To fix this issue https://github.com/OrchardCMS/Orchard/issues/5291

### CODE REVIEW

File: src/Orchard.Web/Modules/Orchard.ContentTypes/Views/Admin/EditPlacement.cshtml [hunk 1]
```diff
@@ -24,11 +24,13 @@
...
-                }
-                catch {
-                }
+				<div class="shape-editor">
+					@try {
+						@Display(Model.AllPlacements[i].Shape)
+					}
+					catch {
+					}
+				</div>
```
Comment (MpDzik) [inline src/Orchard.Web/Modules/Orchard.ContentTypes/Views/Admin/EditPlacement.cshtml]: I think we should avoid tabs and use spaces instead.

Comment (sfmskywalker) [inline src/Orchard.Web/Modules/Orchard.ContentTypes/Views/Admin/EditPlacement.cshtml]: Yes, that's actually our established standard - all spaces, no tabs.

Comment (Hazzamanic) [inline src/Orchard.Web/Modules/Orchard.ContentTypes/Views/Admin/EditPlacement.cshtml]: Right, so I need to replace the tab to four spaces? Sorry, it's my first PR. Also, is there somewhere I can see these "standards"? The docs has a very brief section on C# that doesn't mention spaces from what I can see.

**Comment by sebastienros** [comment]
Nit picking, but yes, please fix the indentation. You can also define it as the default (using spacing when you hit TAB) in the Visual Studio options. This is what we usually do.

**Pull request merged**